### ISSUE TITLE
Bug Report: Templated Used for Management Page

### ISSUE THREAD

**Issue by oliverands** [stcr/subscribe-to-comments-reloaded#258]
Hi, I'm not sure if this is an issue with the plugin or with my configuration.

For some reason, the virtual management page is using a different default page style than the rest of my blog template. You can see my blog here: https://oliverands.com/community/

The management page is adding a left-hand sidebar that the rest of the site does not have. I have tried disabling the virtual management page and creating my own, but the same sidebar content appears.

Can you tell if this is an issue with the plugin? Is it using some page template that is defined in the code instead of making use of my regular page template? Is there some configuration change I can make to address the issue?

Thanks very much!

Todd

**Comment by Reedyseth** [comment]
Hi Todd,

Can you paste the code/content of your template file ??

**Comment by oliverands** [comment]
I'm sorry about this. I determined what the issue was. The management page is calling sidebar.php, and I am using a new custom template which does not include the sidebar. The sidebar.php file was included in the template even though the rest of the template was not making use of it. By removing content from it, I was able to get the management page to display without the sidebar content and to match the rest of my site.

So, it was an issue on my end.

Thanks, again, for the responsiveness to my report and sorry for the inconvenience.
